scripts/update_ci.sh

```diff
@@ -9,18 +8,21 @@ if [ ! -f "template_config.yml" ]; then
 fi
 
 pushd ../plugin_template
-./plugin-template --github {% if ci_update_docs -%} --docs {% endif -%} {{ plugin_name }}
+# Pass on all but the last argument (which is the noissue_marker)
+args="${@:1:$#-1}"
```

Comment:
Can't we just read the noissue marker from the template_config?

Reply:
We probably could, but I did not want to try to parse template_config here.

Reply:
I think we need to, as it could be different on different branches.